hikari/impl/rest.py

```diff
@@ -1593,15 +1592,18 @@ async def delete_messages(
             # I am just gonna invoke these sequentially instead.
             try:
                 if len(pending) == 1:
-                    message = pending.popleft()
+                    message = pending[0]
```

Comment:
can't this still just be `pop(0)`

Reply:
hmm well you can't really be sure that the message wasn't deleted before the error was raised/returned but sure

Reply:
also if the error was NotFoundError cause this was deleting a single message it wasn't a failed delete